test/helper/EventGenerator.ts

```diff
@@ -68,6 +72,24 @@ export function createReactionEvent(targetMessageId: string, reaction: string =
   };
 }
 
+export function createMemberLeaveEvent(conversationId: string, userIds: string[]): MemberLeaveEvent {
+  const conversationQualifiedId = {id: conversationId, domain: ''};
+
+  return {
+    conversation: conversationId,
+    qualified_conversation: conversationQualifiedId,
+    data: {
+      qualified_user_ids: userIds.map(userId => ({id: userId, domain: ''})),
+      reason: MemberLeaveReason.USER_DELETED,
```

Comment:
I think we are missing the part that displays the `x was removed from the team` when this value is set on the event. Am I missing something?